app/controllers/api/v2/users_controller.rb

```diff
@@ -111,6 +111,23 @@ def update
         end
       end
 
+      api :PATCH, "/users/:id/invalidate_jwt", N_("Get vm attributes of host")
+      param :id, String, :required => true
+      description <<-DOC
+        Invalidate JWT for a specific user
+      DOC
+
+      def invalidate_jwt
+        @user = User.find(params[:id])
```

Comment:
```suggestion
        @user = User.where(id: params[:id]).authorized(:edit_user)
```
This will filter only the users that `edit_user` permission applies to them, including custom filters.

Now you will have a set of users that you can safely delete the tokens for:
``` ruby
JwtSecret.where(user_id: @users).destroy_all
```